packages/appstore/src/components/modals/account-type-modal/account-type-modal.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { Button, Modal, DesktopWrapper, MobileDialog, MobileWrapper, UILoader, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { observer } from 'mobx-react-lite';
+import classNames from 'classnames';
+import { useStores } from 'Stores/index';
+import TradigPlatformIconProps from 'Assets/svgs/trading-platform';
+
+type TModalContent = {
+    account_type_card: string;
+    selectAccountTypeCard: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const ModalContent = ({ account_type_card, selectAccountTypeCard }: TModalContent) => {
+    const cardSelection = (cardType: string) => {
+        selectAccountTypeCard(account_type_card === cardType ? '' : cardType);
+    };
+    return (
+        <div className='account-type-card__wrapper'>
+            <div
+                className={classNames('account-type-card', {
+                    'account-type-card--selected': account_type_card === 'Derived',
+                })}
+                onClick={() => cardSelection('Derived')}
+            >
+                <div className='account-type-card__image'>
+                    <TradigPlatformIconProps icon='Derived' size={64} />
+                </div>
+                <div className='account-type-card__header'>
+                    <Text as='h2' weight='bold'>
+                        {localize(`Derived`)}
+                    </Text>
+                </div>
+                <div className='account-type-card__description'>
+                    <Text as='p' size='xxs' align='center'>
+                        {localize(`Trade CFDs on MT5 with Derived indices that simulate real-world market movements.`)}
+                    </Text>
+                </div>
+            </div>
+            <div
+                className={classNames('account-type-card', {
+                    'account-type-card--selected': account_type_card === 'Financial',
+                })}
+                onClick={() => cardSelection('Financial')}
+            >
+                <div className='account-type-card__image'>
```

Comment:
we can declare an array for these 2 cards based on type and add `title`, `description`, `icon`, ... and map them here to prevent duplication

Reply:
Done